### PR TITLE
131356205 Setup HoundCI Integration

### PR DESCRIPTION
**Pull request by andela-aatanda**
Setup HoundCI integration.
That automatically checks conformity and comments in PR.

**What does this PR do?**
Setup for HoundCI Integration

**Description of task to be completed**
Setup HoundCI integration.
That automatically checks conformity and comments in PR.

**How should this be manually tested?**
HoundCI comments in PRs since HoundCI does not have a badge feature for README.md

**What are the relevant pivotal tracker stories?**
#131356205 Integrate HoundCI with badge attached to readme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,31 +18,33 @@
     "postinstall": "node node_modules/grunt-cli/bin/grunt install"
   },
   "dependencies": {
+    "async": "~0.2.9",
+    "bcryptjs": "~1.0.4",
+    "bower": "~1.3.8",
+    "connect-flash": "~0.1.1",
+    "connect-mongo": "~0.3.3",
+    "console-stamp": "~0.1.1",
+    "dotenv": "^2.0.0",
```
Comment (andela-aatanda) [inline package.json]: I have the **dotenv dependency** right here

Comment (q-ode) [inline package.json]: Oh ok!
Didn't see that! #MyBad

But seeing there's no `.env` file what happens?

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,6 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
+mongod
+.env
```
Comment (andela-bmakinwa) [inline .gitignore]: We can have a `.env.sample` file where we put in the env variables and corresponding dummy values. @andela-aatanda

**Comment by q-ode** [comment]
We need to create a PR to staging first.

**Pull request closed without merging**